external/storm-hdfs/src/main/java/org/apache/storm/hdfs/bolt/HdfsBolt.java

```diff
@@ -44,6 +49,25 @@
     private transient FSDataOutputStream out;
     private RecordFormat format;
     private long offset = 0;
+    private static String defaultSourceDir = "/tmp/source";
+    private static String defaultDestDir = "/tmp/dest";
+    private static String defaultFileExtension = ".txt";
+
+
+    public HdfsBolt() {
+      this(defaultSourceDir, defaultDestDir, defaultFileExtension);
+    }
+
+    public HdfsBolt(String defaultSourceDir, String defaultDestDir, String fileExtension) {
+      this.withRotationPolicy(new TimedRotationPolicy(1.0f, TimedRotationPolicy.TimeUnit.MINUTES))
+          .withConfigKey("hdfs.config")
+          .withRecordFormat(new DelimitedRecordFormat().withRecordDelimiter("|"))
+          .withFileNameFormat(new DefaultFileNameFormat()
+                  .withPath(defaultSourceDir)
+                  .withExtension(fileExtension))
+          .withSyncPolicy(new CountSyncPolicy(1000))
+          .addRotationAction(new MoveFileAction().toDestination(defaultDestDir));
```

Comment:
I would like to see it that if there is no stagingDir specified that the destDir is used for the DefaultFileNameFormat path and there is no MoveFileAction.  If there is a  stagingDir then it behaved like this.